test/chi/components/copy-text.pug

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Shoudn't that file contain HTML representation of this component ?

Reply:
Yeah!! good catch